src/utils/gaa.js

```diff
@@ -292,9 +292,13 @@ export let GoogleUserDef;
 /**
  * Returns true if the query string contains fresh Google Article Access (GAA) params.
  * @param {string} queryString
+ * @param {boolean} shouldValidateAccessType
  * @return {boolean}
  */
-export function queryStringHasFreshGaaParams(queryString) {
+export function queryStringHasFreshGaaParams(
+  queryString,
+  shouldValidateAccessType = true
```

Comment:
Could we update this name to `allowAllAccessTypes`? I think that'd be a bit clearer

Reply:
Done